docs: remove/rename major elt references

sbalnojan opened the pull request:
As discussed in https://github.com/meltano/meltano/issues/7052, I removed/moved a few of the major mentions of the "meltano elt" command, making "meltano run" the prime suspect for doing your first bigger things with Meltano.

I also moved a few of the references to running Singer taps in isolation etc. to the advanced topics. Those IMHO shouldn't be inside the "Replicate Data" area. That should focus just on Meltano, and should be inclusive to e.g. airbyte taps.

sbalnojan committed:
docs: remove/rename major elt references

pre-commit-ci[bot] committed:
[pre-commit.ci] auto fixes from pre-commit.ci hooks

for more information, see https://pre-commit.ci

WillDaSilva commented on docs/src/_guide/integration.md:
```diff
@@ -272,28 +245,12 @@ meltano invoke <plugin> [PLUGIN_ARGS...]
 In this case, it can be useful to run the tap with just the single stream selected.
 
-Instead of duplicating the extractor in `meltano.yml`, try running `meltano elt` with the [`--select` flag](/reference/command-line-interface#parameters-2).
+Instead of duplicating the extractor in `meltano.yml`, try running `meltano run` with the [`--select` flag](/reference/command-line-interface#parameters-2).
```
`meltano run` has no `--select` flag.

WillDaSilva commented on docs/src/_guide/integration.md:
```diff
@@ -237,12 +210,12 @@ This command returns the merged result of the latest completed state plus any ne
 Unlike [`meltano state merge`](/reference/command-line-interface#state),[`meltano state set`](/reference/command-line-interface#state) will insert a complete record, which causes meltano to ignore any previous state records, whether completed or partial.
 
-Note that if you already have a state file you'd like to use, it can be provided explicitly using [`meltano elt`](/reference/command-line-interface#elt)'s `--state` option or the [`state` extractor extra](/concepts/plugins#state-extra).
+Note that if you already have a state file you'd like to use, it can be provided explicitly using [`meltano run`](/reference/command-line-interface#run)'s `--state` option or the [`state` extractor extra](/concepts/plugins#state-extra).
```
`meltano run` has no `--state` option.

tayloramurphy commented on docs/src/_guide/integration.md:
```diff
@@ -37,12 +37,12 @@ On a shell, you can explicitly `export` environment variables, that will be pass
+To verify that these environment variables will be picked up by Meltano as you intended, you can test them with [`meltano config <plugin>`](/reference/command-line-interface#config) before running `meltano run`.
 
 If you're using [`meltano schedule`](/reference/command-line-interface#schedule) to [schedule your pipelines](/guide/orchestration), you can specify environment variables for each pipeline in your [`meltano.yml` project file](/concepts/project#meltano-yml-project-file), where each entry in the `schedules` array can have an `env` dictionary:
 
```
@sbalnojan just below this is the non-job format for schedules, would you want to update that as well? The current format uses `elt` behind the scenes while the jobs format would use `run`.

tayloramurphy commented on docs/src/_guide/integration.md:
````diff
@@ -37,12 +37,12 @@ On a shell, you can explicitly `export` environment variables, that will be pass
 ```bash
 export TAP_FOO_BAR=bar
````
Perhaps this is best for another issue / PR, but we probably shouldn't recommend folks manage env vars this way. They can use the `env:` key to export environment variables!

sbalnojan committed:
remove overhead completely.

sbalnojan commented:
@tayloramurphy @WillDaSilva thanks! I made the changes. I just cut out a bit of these parts completely, I think for a "replicate data" guide the "pipeline specific configuration" is waaaay too much overhead, so it's now inside the advanced topics (those get way less traffic so I didn't bother to rewrite the schedule syntax for now).

tayloramurphy commented:
@sbalnojan it looks like Will's comments still need addressing around the available options for run vs elt.

sbalnojan committed:
Merge branch 'docs-7052-elt-rm' of https://github.com/meltano/meltano into docs-7052-elt-rm

sbalnojan committed:
rename

pre-commit-ci[bot] committed:
[pre-commit.ci] auto fixes from pre-commit.ci hooks

for more information, see https://pre-commit.ci

sbalnojan committed:
redo schedule refs

sbalnojan committed:
Merge branch 'docs-7052-elt-rm' of https://github.com/meltano/meltano into docs-7052-elt-rm

sbalnojan commented:
@tayloramurphy hm so now everything should be addressed, I think I removed all parts referenced by Will, and I incorporated the comments from @pnadolny13 on the ticket.  If something is still missing, please highlight the line, otherwise I won't be able to find it :-)

sbalnojan committed:
Merge branch 'main' into docs-7052-elt-rm

WillDaSilva commented on docs/src/_guide/integration.md:
```diff
@@ -15,69 +15,17 @@ and keeps track of [the incremental replication state](#incremental-replication-
 ## Plugin configuration
 
-As described in the [Configuration guide](/guide/configuration#configuration-layers), [`meltano elt`](/reference/command-line-interface#elt) will determine the configuration of the extractor, loader, and (optionally) transformer by looking in [**the environment**](/guide/configuration#configuring-settings), your project's [**`.env` file**](/concepts/project#env), the [system database](/concepts/p ...
+As described in the [Configuration guide](/guide/configuration#configuration-layers), [`meltano run`](/reference/command-line-interface#run) will determine the configuration of the extractor, loader, and (optionally) transformer by looking in [**the environment**](/guide/configuration#configuring-settings), your project's [**`.env` file**](/concepts/project#env), the [system database](/concepts/p ...
```
The configuration layers section of the configuration guide describes these things for `meltano elt`, not `meltano run`. Even if it were updated to say `meltano run` instead of `meltano elt`, I'm not sure that that would make it true.

WillDaSilva replied:
@sbalnojan Two possible quick fixes:
- Update `guide/configuration#configuration-layers` to use `meltano run` instead of `meltano elt` (preferable).
- Remove the link to `guide/configuration#configuration-layers`. The rest of the paragraph seems mostly fine (if a bit light on details) without it.

In either case you'll need to rephrase this part:
> will determine the configuration of the extractor, loader, and (optionally) transformer

sbalnojan commented:
@WillDaSilva I will close it.
